impl/src/main/java/org/jboss/weld/bean/builtin/BeanManagerProxy.java

```diff
@@ -272,4 +273,13 @@ public BeanManagerImpl unwrap() {
         delegate().disposeAnnotatedType(type, id);
     }
 
+    @Override
+    public boolean isContextActive(Class<? extends Annotation> scopeType) {
+        return delegate().isContextActive(scopeType);
+    }
+
+    @Override
+    public Collection<Class<? extends Annotation>> getScopes() {
```

Comment:
Should this be documented?

Reply:
Yes, but not quite yet - what needs documenting is Weld API and we are not sure about it's final form yet. But granted, we need a backing JIRA to keep in mind to document all API changes once we settle them - I'll create that.

Reply:
Makes sense, thanks.